Source/ProgressionSystemRuntime/Public/LevelActors/PSStarActor.h

```diff
@@ -9,21 +9,39 @@ UCLASS()
 class PROGRESSIONSYSTEMRUNTIME_API APSStarActor : public AActor
 {
 	GENERATED_BODY()
-	
-public:	
+
+public:
 	// Sets default values for this actor's properties
 	APSStarActor();
 
 	UFUNCTION()
 	void SetStaticMesh(UStaticMesh* Mesh);
-	
+
 protected:
 	// Called when the game starts or when spawned
 	virtual void BeginPlay() override;
+
+	/** Function called every frame on this Actor */
+	virtual void Tick(float DeltaTime) override;
+	/** Static mesh of actor */
 	UPROPERTY()
 	TObjectPtr<class UStaticMeshComponent> MeshComponentInternal = nullptr;
```

Comment:
Please remove `SetStaticMesh` and `MeshComponentInternal` as redundant,as I remember its functionality is fully set in actor now

Reply:
removed as part of https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/5d72f8876997aaa97e4601fe31363fec4893004e